src/R/Components/Impl/ConnectionManager/Implementation/ViewModel/ConnectionStatusBaseViewModel.cs

```diff
@@ -30,19 +30,13 @@ internal abstract class ConnectionStatusBaseViewModel : BindableBase, IDisposabl
             IsRunning = connectionManager.IsRunning;
         }
 
-        public virtual void Dispose(bool disposing) {
-            _disposableBag.TryDispose();
-        }
-
-        public void Dispose() {
-            Dispose(true);
-        }
+        public virtual void Dispose(bool disposing) => _disposableBag.TryDispose();
```

Comment:
Actually, you can look into DevDiv.ruleset we have checked in, maybe it is gone?